Makefile

```diff
@@ -98,6 +98,24 @@ run-dev-image:
 		-v $(CURDIR)/collectd:/usr/src/collectd:cached \
 		signalfx-agent-dev /bin/bash
 
+.PHONY: run-k8s-tests
+run-k8s-tests:
+	docker rm -f -v minikube registry || true
+	docker run --rm -it \
+		$(extra_run_flags) \
+		--net host \
+		-v $(CURDIR):/go/src/github.com/signalfx/signalfx-agent:cached \
+		signalfx-agent-dev \
+			pytest \
```

Comment:
Make this just the pytest command without all the docker stuff.  That way it can be run straight in the dev container without modification.

Reply:
But leave the minikube/registry creation stuff though.